_episodes/12-cluster.md

```diff
@@ -23,143 +27,278 @@ On the first day, her supervisor greets her friendly and welcomes her to the job
+{: .callout}
+
+> ## Relative to your Laptop
+> 
+> Note down the number of CPU cores, the amount of RAM and the total disk space available on `{{ site.workshop_login_host }}`. Compare it to your laptop!
+> 
+> Bonus: Divide the values obtained from `{{ site.workshop_login_host }}` by the numbers obtained for your laptop. How much more powerful is the login node of the cluster compared to your laptop?
 {: .challenge}
 
-> ## Explore The Head Node
->
-> Now we'll compare the size of your computer with the size of the head node: To see the number of
-> processors, run:
->
-> ```
-> nproc --all
-> ```
-> {: .bash}
->
-> or
+
+## Transferring Data
+
+The admin continues to explain, that typically people perform computationally heavy tasks on the cluster and prepare files that contain the results or a subset of data to create final results on the individuals laptop. So communication to and from the cluster is done mostly by transferring files. For example, Lola is asked to use a [file of her liking]({{page.root}}/filesystem/home/admin/this_weeks_canteen_menus/todays_canteen_menu.pdf) and transfer it over. For this, he advises her to use the secure copy command, `scp`. As before, this establishes a secure encrypted temporary connection between Lola's laptop and the cluster just for the sake of transferring the files. After the transfer has completed, scp will close the connection again.
+
+~~~ 
+$ scp todays_canteen_menu.pdf lola@{{ site.workshop_login_host }}:todays_canteen_menu.pdf
+~~~
+{: .language-bash}
+
+~~~ 
+todays_canteen_menu.pdf                                              100%   28KB  27.6KB/s   00:00
+~~~
+{: .output}
+
+She can now `ssh` into the cluster again and check, if the file has arrived after she just uploaded it:
+
+~~~ 
+$ ssh lola@{{ site.workshop_login_host }}
+Last login: Tue Mar 14 14:17:44 2017 from lolas_laptop
+-bash-4.1$ ls
+~~~
+{: .language-bash}
+
+~~~ 
+todays_canteen_menu.pdf
+~~~
+{: .output}
+
+Now, let's try the other way around, i.e. downloading a file from the cluster to Lola's laptop. For this, Lola has to swap the two arguments of the `scp` command she just issued.
+
+~~~ 
+$ scp lola@{{ site.workshop_login_host }}:todays_canteen_menu.pdf todays_canteen_menu_downloaded.pdf
+~~~
+{: .language-bash}
+
+Lola notices how the command line changed. First, she has to enter the source (`lola@{{ site.workshop_login_host }}`) then put a `:` and continue with the path of the file she wants to download. After that, separated by a space, the destination has to be provided, which in this case is a file `todays_canteen_menu_downloaded.pdf` in the current directory.
```

Comment:
`path of the file she wants` with respect to what ?  I've seen user confused because they 

```
local $ sshremote
remote $ cd folder
remote $ ls 
foo.png
remote $ logout
local $ scp remote:foo.png .  # does not work, ssh doe not "remember" you were in `folder`
```

Reply:
added a callout on this with a02a3e5. Once this PR is merged, I'd love to ask you to rewrite the above into a challenge and submit a PR. If it happens in the wild, it's perfect for a challenge.